### PR TITLE
[bugfix/527] Prevents oculus version from launching if the original oculus version is not present

### PR DESCRIPTION
**Pull request by LiamillionSS**
Closes https://github.com/Zagrios/bs-manager/issues/527

Runs check for original oculus version when attempting to run version downloaded from BSM and throws BSLaunchError if original not found. Also updated "must-be-installed-once" error.

Closes https://github.com/Zagrios/bs-manager/issues/655 while adding translated messages.

### CODE REVIEW

**Comment by LiamillionSS** [comment]
I dont know why my comments didnt save so here we again:

Closes #527

Runs check for original oculus version when attempting to run version downloaded from BSM and throws BSLaunchError if original not found. Also updated "must-be-installed-once" error. 

Included close for #655 when adding translation messages.

**Comment by LiamillionSS** [comment]
Force-push to clean up commits, no changes to actual code content from before

File: assets/jsons/translations/en.json [hunk 1]
```diff
@@ -455,7 +456,8 @@
                     "EXIT": "BeatSaber stopped abruptly, tries to check the files.",
                     "OCULUS_LIB_NOT_FOUND": "Check that the Oculus application is properly installed, and that the libraries are correctly defined in Oculus.",
                     "PROTON_NOT_SET": "Set the Proton path in settings.",
-                    "PROTON_NOT_FOUND": "Set the Proton path in settings."
+                    "PROTON_NOT_FOUND": "Set the Proton path in settings.",
+                    "ORIGINAL_OCULUS_NOT_INSTALLED": "Download the original Beat Saber from Oculus before opening."
```
Comment (Zagrios) [inline assets/jsons/translations/en.json:460]: I wonder if `Beat Saber must be installed from the Oculus app before opening` would be better 🤔

**Review by Zagrios — CHANGES_REQUESTED** [review]

File: assets/jsons/translations/en.json (hunk 1 above)
Comment (LiamillionSS) [reply assets/jsons/translations/en.json]: Yep, agreed

**Review by Zagrios — APPROVED** [review]

**Comment by Zagrios** [comment]
Thanks a lot @LiamillionSS ! ❤️ 
Merging 🚀

**Pull request merged**